.gitignore

```diff
@@ -7,6 +7,7 @@ target
 .ear
 .ejb
 *.iml
+*.ipr
```

Comment:
Looks like the .ipr was created by intellij. Removed and added to gitignore instead.

Reply:
👍